elixir-ci/1.15.2-erlang-26.0.2-debian-bullseye-20230612-slim/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM hexpm/elixir:1.15.2-erlang-26.0.2-debian-bullseye-20230612-slim
+
+RUN apt install -y --no-install-recommends curl python3 py3-pip postgresql13-client jq nodejs~=16 yarn bc git build-base imagemagick brotli chromium bash openssh-client docker-cli xvfb libc6-compat tar
+
+RUN apk upgrade --no-cache
+
+RUN pip install -U pip
+
+RUN pip install awscli virtualenv requests boto3 --upgrade
+
+RUN mix local.hex --force
+RUN mix local.rebar --force
```

Comment:
Most of this is stuff is not used by the accounts service but I left it in case we wanna use it for the platform in the near future